index.html

```diff
@@ -212,6 +212,24 @@ <h3>MKV to MP4</h3>
     </div>
     <!-- ends MKV to MP4 -->
 
+    <!-- Rewrap DV -->
+    <label class="recipe" for="rewrap-dv">Rewrap DV video to .dv file</label>
+    <input type="checkbox" id="rewrap-dv">
+    <div class="hiding">
+      <h3>Rewrap DV video to .dv file</h3>
+      <p><code>ffmpeg -i <i>input_file</i> -f rawvideo -c:v copy <i>output_file.dv</i></code></p>
+      <p>This script will take a video that is encoded in the <a href="https://en.wikipedia.org/wiki/DV">DV Codec</a> but wrapped in a different wrapper (such as MOV) and rewrap it into a raw DV file (with the .dv extension). Since DV files potentially contain a great deal of provenence metadata within the DV stream, it is necessary to rewrap files in this method to avoid unintentional stripping of this metadata.</p>
```

Comment:
• provenance rather than provenence  
• "different wrapper" could be "different container", just to avoid saying wrapper three times in the sentence. That's total nitpicking though!